ignite/templates/app/files/cmd/{{binaryNamePrefix}}d/cmd/root.go.plush

```diff
@@ -121,13 +123,38 @@ func NewRootCmd() *cobra.Command {
 		flags.FlagKeyringBackend: "test",
 	})
 
+	autoCliOpts, err := enrichAutoCliOpts(autoCliOpts, clientCtx)
```

Comment:
I don't think it does something, as depinject would already inject those fields in the AutoCLI options.

I think this is more related to the unreleased fixes we have for client/v2:

- https://github.com/cosmos/cosmos-sdk/issues/19530
- https://github.com/cosmos/cosmos-sdk/pull/19646